rpcs3/Emu/RSX/Common/TextureUtils.cpp

```diff
@@ -145,6 +145,33 @@ struct convert_16_block_32
 		}
 	}
 };
+
+struct convert_16_block_32_swizzled
```

Comment:
There are too many mac workarounds here, making this file difficult to follow. Some refactoring will be required later, but we can leave that alone for now as future work.